server/src/guest/guest.service.ts

```diff
@@ -42,15 +42,6 @@ export class GuestService {
 		return await this.guestRepository.findOneBy({ uuid });
 	}
 
-	/**
-	 * Get the primary data and associate arrays of a guest based on the given uuid
-	 * @param uuid A uuid belonging to an existing user
-	 * @returns The primary data and associate arrays of that user
-	 */
-	async getGuest(uuid: string): Promise<primaryData> {
```

Comment:
Do we just not need this anymore?

Reply:
It turned into a duplicate of getPrimaryData so no need anymore